_docs/customer-stories/epaairnow.md

```diff
@@ -0,0 +1,55 @@
+---
+parent: customer-stories
+layout: docs
+sidenav: true
+title: Environmental Protection Agency AirNow
+kicker: Case study
+redirect_from:
+  - /overview/customer-stories/environmental-protection-agency-airnow/
+  - /case-studies/epaairnow/
+---
+
+!["Screenshot from analytics.usa.gov on June 13, 2023"]({{ site.baseurl }}/assets/images/content/TopDomainsAirNow.png)
```

Comment:
```suggestion
!["Screenshot from analytics.usa.gov showing the top requested government domains for the last 7 days  on June 13, 2023"]({{ site.baseurl }}/assets/images/content/TopDomainsAirNow.png)
```

Alt text for images should be as descriptive as possible since it will used by people using [screen readers](https://developer.mozilla.org/en-US/docs/Glossary/Screen_reader) to understand the contents of an image. Alt text is also used in case the image can't be loaded for any reason, so it should explain the image contents well.